{{cookiecutter.repostory_name}}/README.md

```diff
@@ -67,28 +67,28 @@ mkdir {{ cookiecutter.repostory_name }}
 ```sh
 # locally
 git remote add production root@<server>:~/repos/{{ cookiecutter.repostory_name }}.git
-git push production master
+git push production main
 ```
 
 ```sh
 # remote server
 cd ~/repos/{{ cookiecutter.repostory_name }}.git
 
-cat <<EOT >> hooks/post-receive
+cat <<'EOT' > hooks/post-receive
```

Comment:
'' change doesn't do much
>>
> 
does

Reply:
It actually does. Without quoting it, all variables are evaluated. Red content is produced without the quotes, green with the quotes:
```diff
<     if [[  =~ .*/main$ ]]; then
<         export GIT_DIR="/repos/.git/"
<         export GIT_WORK_TREE="/domains//"
---
>     if [[ $ref =~ .*/main$ ]]; then
>         export GIT_DIR="$ROOT/repos/$REPO.git/"
>         export GIT_WORK_TREE="$ROOT/domains/$REPO/"
11c11
<         cd
---
>         cd $GIT_WORK_TREE
```

Reply:
didn't know that, thanks :)